lib/Controller/ExAppsPageController.php

```diff
@@ -451,6 +451,7 @@ public function enableApps(array $appIds, array $groups = []): JSONResponse {
 						}
 					} else {
 						$this->logger->error(sprintf('Failed to deploy %s ExApp', $appId));
+						$this->exAppService->removeExAppL10NFolder($appId);
```

Comment:
we do not need this, extraction of `L10N` folder should be during "Registration" step at the end